Models/movieModel.js

```diff
@@ -1,142 +1,103 @@
-const mongoose = require('mongoose');
-const fs = require('fs');
-const validator = require('validator');
+const mongoose = require("mongoose");
```

Comment:
Is there any significant difference between using double quotes and single quotes